src/wordlist.c

```diff
@@ -1076,7 +1076,7 @@ void do_wordlist_crack(struct db_main *db, const char *name, int rules)
 			log_event("- %d preprocessed word mangling rules", rule_count);
 	}
 
-	if (dupeCheck || rules) {
+	if ((dupeCheck || rules) && !status.suppressor_start_time) {
```

Comment:
This may be unreliable - I think `status.suppressor_start_time` may just happen to be zero if the suppressor got enabled so early at startup that the time didn't advance by a single unit yet.

I think it's better to drop this check and instead make the added check in `suppressor_init` the official and important one, not just defensive programming.

Reply:
I dropped the wordlist commit now.